lib/users.test.js

```diff
@@ -50,3 +50,79 @@ describe('Validates user signup', () => {
     })
   })
 })
+
+jest.mock('../sequelize/db')
+const bcrypt = require('bcrypt')
+const db = require('../sequelize/db')
```

Comment:
move these to the top of the file. Other user functions may need to use these. All mocks and requires must be at the top of the file.